src/main/java/ladder/model/Results.java

```diff
@@ -0,0 +1,31 @@
+package ladder.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Results {
+
+    private final Participants participants;
+    private final List<String> results;
```

Comment:
List<String> results 게임 결과는 사용자에게 입력 받은 값을 그대로 출력해주는 역할 그 이외의 것을 하지 않아
일급컬렉션을 만드는 대신 List로 필드 선언을 했습니다. 일급 컬렉션으로 만드는 게 더 나은 선택이었을까요?
아직 기준이 모호한 듯 합니다.. ㅎㅎ

Reply:
일급 컬렉션은 둘째치고, results가 가지는 `List<String>`이 어떤 의미를 나타내는지 의도를 전달하는 게 더 중요해 보여요.

이 경우 일급 컬렉션을 만들지 않았다면, `List<Result>`와 같이 타입을 명시적으로 나타내는 게 좋지 않았을까요?

Reply:
어제 미팅에서 잘 설명해주셔서 감사합니다👍
굳이 변경을 안했어도 되나 싶었지만, 미션을 통해 학습하는 만큼 좀 더 명확히 할 수 있도록 했습니다.
 
```suggestion
    private final List<Result> results;
```

> 기존의 문제점은 **결과값이 무엇을 의미하는지에 대해 컨텍스트를 코드만 보고 추론하기 어렵다**는 점.
> 명시적인 타입 `List<Result>` 사용해서 단순 텍스트(String)이 아니라 **게임 결과**란 값을 가진 객체로 의미를 부여했습니다. 
> 이로써 결과에 대한 유효성 검증을 객체 내에서 할 수 있고, 정의를 더 상세하게 할 수 있게 개선되었습니답!_!